vtex-apps/store-components
# Update CSS handles on InfoCard component

#### What problem is this solving?

Review the available CSS handles on the InfoCard component, add a new `infoCardCallActionText` handle and also use `vtex.css-handles` app to resolve those handles.

#### How should this be manually tested?

You can inspect the `InfoCard` at the home page and see that the CSS classes being added are the same as the ones before.

[Workspace](https://victorhmp--storecomponents.myvtex.com/)

#### Checklist/Reminders


#### Screenshots or example usage

#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
✔️ | Technical improvements 

#### Notes
